web/src/views/DandisetLandingView/DandisetLandingView.vue

```diff
@@ -172,6 +151,14 @@ export default {
       },
     },
   },
+  async created() {
+    const schema = SCHEMA;
+
+    // TEMPORARY: Remove known circular reference
```

Comment:
Is this a circular reference in the schema itself, or in some data that is being used? Do all circular references need to be removed like this? Is there a solution that will deprecate this temporary fix?

Reply:
It's the schema itself. See [this](https://github.com/dandi/dandi-core/pull/4#discussion_r452352431) comment. Yes, all circular references need to be removed, as otherwise `Ajv` can't run validation. However, this appears to be the only circular reference in this schema. I'm not sure of the long-term solution.